utils/loaddata_utils.py

```diff
@@ -54,3 +56,27 @@ def create_loaddata_illum_csv(
     command = f"pe2loaddata --index-directory {index_directory} {config_path} {path_to_output} --illum --illum-directory {illum_directory} --plate-id {plate_id} --illum-output {illum_output_path}"
     os.system(command)
     print(f'{illum_output_path.name} is created!')
+
+def edit_loaddata_csv(path_to_loaddata_csv:pathlib.Path):
+    """
+    This function loads in the loaddata csv and edit it to remove unnessecary rows and 
```

Comment:
Can you add a note here about _why_ rows are unnecessary? It will also be good to point to the file Erin shared to credit inspiration

Reply:
I added more documentation into the README and added the image.sc post link. 

To add it here, the rows are unnecessary because they are the rows that are not the last z-plane. Since the max projected images have the name and the metadata of the last z-plane, we only need rows with the last z-plane.